### PR TITLE
Support char type

### LINKED ISSUE #103: Support for `char`
👋 Thanks for building and open sourcing typeshare, I'm just trying it out for the first time and I've noticed the `char` type get's output as `char`, should/could this be output as `string`?

### PR DESCRIPTION
**Pull request by czocher**
Add support for the Rust `char` type.

Closes #103

### CODE REVIEW

**Comment by Lucretiel** [comment]
Looks great, just a small Swift nit and it's good to merge

**Comment by czocher** [comment]
@Lucretiel already fixed ;).

**Comment by Lucretiel** [comment]
I'm not seeing it on the PR; does it need to be pushed?

**Comment by czocher** [comment]
> I'm not seeing it on the PR; does it need to be pushed?

@Lucretiel You mean the change requested by @CerulanLumina? I amended the change just after they noticed it. This is the current version that shows up when I check the changes in this PR: https://github.com/1Password/typeshare/pull/128/files#diff-e3f812823b9b43de87bbd7c54ab61d3e102c387d74245c65cd8a0e3bbca2193e, is there some other thing I missed?

**Comment by Lucretiel** [comment]
Whoops, it was *I* who didn't didn't press Push on my review and just let it sit as a draft

**Review by Lucretiel — CHANGES_REQUESTED** [review]

File: core/src/language/swift.rs [hunk 1]
```diff
@@ -190,6 +190,7 @@ impl Language for Swift {
                 "CodableVoid".into()
             }
             SpecialRustType::String => "String".into(),
+            SpecialRustType::Char => "UnicodeScalar".into(),
```
[the remark this answers is not in the document]
Comment (czocher) [reply core/src/language/swift.rs:193]: Thanks!

**Comment by czocher** [comment]
@Lucretiel done!

**Comment by CerulanLumina** [comment]
Looks good! Just need the commit to pass signing then we'll merge!

**Review by Lucretiel — APPROVED** [review]

**Comment by czocher** [comment]
There was some weird mixup with my GitHub account configuration, reuploaded my GPG key and it seems to work. Thanks!

**Pull request merged**